.github/workflows/build_dockers_workflow.yml

```diff
@@ -20,7 +20,7 @@ name: Build docker images - workflow_call/on-demand
         default: x86-64
       features:
         type: string
-        default: 'safe,grpc'
+        default: 'safe,libtor,metrics'
```

Comment:
Should we not keep grpc?
```libtor``` is the default for nix, but having it in seems more clear.

Reply:
I think we do need grpc for the block explorers

Reply:
Docker images for tari_suite.

Console wallet has the empty feature too. Was it not meant as a none default build feature?

Reply:
I did local build with and without grpc feature, and my base node still started up, so I think this can go in.